flair/data.py

```diff
@@ -337,19 +337,139 @@ def __repr__(self) -> str:
         )
 
 
+def space_tokenizer(text: str) -> List[Token]:
+    """
+    Tokenizer based on space character only.
+    """
+    tokens: List[Token] = list()
+    word = ""
+    index = -1
+    for index, char in enumerate(text):
+        if char == " ":
+            if len(word) > 0:
```

Comment:
`if word` instead ?

Reply:
Is it more readable? Shorter but it requires to know some Python convention. I have not found any other occurrence of this pattern but I may have missed something.